src/test_dsAudio_parse_configuration.c

```diff
@@ -167,22 +168,37 @@ int test_dsAudio_parse_configuration()
         snprintf(key_string, DS_AUDIO_KVP_SIZE, "dsAudio/Ports/%d/ATMOS_Capabilities" , i+1);
         gDSAudioPortConfiguration[i].atmos_capabilites = ut_kvp_getUInt16Field( ut_kvp_profile_getInstance(), key_string  );
 
+        // comment code due to bug in : ut_kvp_fieldPresent unable to parse the %d
+        //if(ut_kvp_fieldPresent(ut_kvp_profile_getInstance(),"dsAudio/Ports/%d/dialog_enhancement_level"))
```

Comment:
check can be done as this:
snprintf(key_string, DS_AUDIO_KVP_SIZE, "dsAudio/Ports/%d/dialog_enhancement_level" , i+1);
if(ut_kvp_fieldPresent(ut_kvp_profile_getInstance(),"key_string"))

Reply:
Updated

Reply:
updated the validation